src/client/app/index.ts

```diff
@@ -1,18 +1,26 @@
 import { ProblemRunner as Component } from '../component/ProblemRunner';
-import { loadProblem } from './comm';
+import { Content } from '../model/Content';
+import { Problem } from '../model/Problem';
+import { CONTENT_CLASS, loadEntity, PROBLEM_CLASS } from './comm';
+
+function prepareURL(problemId: string): string {
+    return '/problem/' + encodeURI(problemId).replace(/\//g, '%2f');
```

Comment:
Why can we have slashes in the problemId?

Reply:
Because some problem identifiers were written with "w/" and "w/o".
That might not be blocked in the future but I wanted to give a try for a fix that `encodeURI` and `escape` let pass through...

Reply:
ok